README.md

```diff
@@ -1 +1,10 @@
-# HikingJournal
\ No newline at end of file
+# HikingJournal
```

Comment:
I like this way of headings creation (instead of [alternate syntax](https://www.markdownguide.org/basic-syntax#alternate-syntax)).